### DISCUSSION TITLE
Problems inserting new data into a one-to-many value relationship in Qfield 2022.

### DISCUSSION THREAD

**Discussion by Christmannn** [opengisch/QField#2993] [Q&A]
Since the last update of Qfield and the change of Android privacy policies, Qfield works badly. I have a Qfield project with a one to many table relationship. One layer with point geometry (point) and one without geometry (review). When inserting a new data in the point table, I would have a list to see the related data of the revision table. A plus (+) appeared in the list to insert the new data into the related table. With the new update the plus (+) has disappeared. The project opens perfectly in qfield and I can insert data to all layers with no problem but I can't insert new data through the parent layer to the related table. Attached image.
The project in Qgis works perfectly. If I open the Qfield project in the desktop version of Qfield it works perfectly. The problem is in the Qfield of the mobile.

![WhatsApp Image 2022-06-20 at 10 48 42 AM](https://user-images.githubusercontent.com/107842743/174564383-07105aea-c537-4d7f-8bba-253c0e2ee74e.jpeg)

![Captura](https://user-images.githubusercontent.com/107842743/174564340-77e26fa8-c68d-423c-ba1f-6ad4e662c258.PNG)

Thank you very much for the help.

**Comment by nirvn** [comment]
@Christmannn , can you attach a test project here?

**Reply by Christmannn** [reply]
Of course. Thanks a lot.


[sim.zip](https://github.com/opengisch/QField/files/8939733/sim.zip)

**Reply by nirvn** [reply]
@Christmannn , you spotted a bug, well done! :wink: thanks for the test project, made it super easy to fix the regression. You can confirm your issue is gone by trying the APKs in this PR (https://github.com/opengisch/QField/pull/3000).

**Reply by Christmannn** [reply]
Hello nirvn, effectively, the problem is already solved!. I detected that when using autogenerate within the attributes form, the button to insert new data in the relation did not work. However, it does work when you use drag and drop designer. You must add the relationship to the drag and drop designer.
![Captura](https://user-images.githubusercontent.com/107842743/174958289-6fc17465-489f-4f9b-bf4e-7d6a14d21a13.PNG)

**Reply by nirvn** [reply]
@Christmannn , right, you have found a good workaround. Yet, QField should (and now does) handle auto-generated forms too :)
